test/ui-test/testSuites/suite_wallet/tst_wallet/test.feature

```diff
@@ -2,56 +2,65 @@ Feature: Status Desktop Wallet
 
-    Background: Sign up & Enable wallet section
-        Given A first time user lands on the status desktop and generates new key
-        When user signs up with username "tester123" and password "TesTEr16843/!@00"
-        Then the user lands on the signed in app
-        When the user opens app settings screen
-        When the user activates wallet and opens the wallet section
-        When the user accepts the signing phrase
+   	The feature start sequence is the following (setup on its own `bdd_hooks`):
 
+    ** given A first time user lands on the status desktop and generates new key
+    ** when user signs up with username "tester123" and password "TesTEr16843/!@00"
+    ** and the user lands on the signed in app
 
-	Scenario:  User can observe an account data
-        When the user opens app settings screen
+    ** given the user opens app settings screen
+    ** and the user activates wallet and opens the wallet section
+    ** and the user accepts the signing phrase
+
+    Background: Navigation to main wallet screen
+
+        Given the user opens wallet screen
+        And the user clicks on the first account
+
+	Scenario: The user can import seed phrase and observe an account data
+        Given the user opens app settings screen
         And the user opens the wallet settings
         And the user toggles test networks
         And the user opens wallet screen
-        And the user imports a seed phrase with one and TesTEr16843/!@00 and pelican chief sudden oval media rare swamp elephant lawsuit wheat knife initial
-        Then the new account one is added
-        And the user has a positive balance of ETH
-        And the user has a positive balance of STT
-        # And the collectibles are listed for the one
-        And the transactions are listed for the added account
-
-
-	Scenario: User can manage a list of accounts
-        When the user adds watch only account with AccountWatch and 0x8397bc3c5a60a1883174f722403d63a8833312b7
-        Then the new account AccountWatch is added
-        When the user imports a private key with AccountPrivate and TesTEr16843/!@00 and 8da4ef21b864d2cc526dbdb2a120bd2874c36c9d0a1fb7f8c63d7f7a8b41de8f
-        Then the new account AccountPrivate is added
-        When the user imports a seed phrase with AccountSeed and TesTEr16843/!@00 and pelican chief sudden oval media rare swamp elephant lawsuit wheat knife initial
-        Then the new account AccountSeed is added
-        When the user generates a new account with AccountGenerated and TesTEr16843/!@00
-        Then the new account AccountGenerated is added
-        When the user deletes the account AccountGenerated
-        Then the account AccountGenerated is not in the list of accounts
-        When the user opens app settings screen
+        When the user imports a seed phrase with "one" and "TesTEr16843/!@00" and "pelican chief sudden oval media rare swamp elephant lawsuit wheat knife initial"
```

Comment:
Revise for clarity as it's not necessarily clear what the parameters are for. Suggestion:

```suggestion
        When an account named "one" is added via imported seed phrase "pelican chief sudden oval media rare swamp elephant lawsuit wheat knife initial", authenticated using password "TesTEr16843/!@00" 
```